rust/worker/src/blockstore/arrow/blockfile.rs

```diff
@@ -373,6 +523,81 @@ mod tests {
         }
     }
 
+    #[parameterized(operation = {"gt", "gte", "lt", "lte"})]
```

Comment:
i need to research the adoption of this library

Reply:
we should just be able to use proptest! with an enum here and get the rng from proptest